lib/babel.js

```diff
@@ -13,7 +12,7 @@ var testPath = process.argv[2];
 var babel;
 
 try {
-	var localBabel = resolveFrom('.', 'babel-core') || resolveFrom('.', 'babel');
+	var localBabel = resolveFrom('.', 'babel-core');
 	babel = require(localBabel);
 } catch (err) {
 	babel = require('babel-core');
```

Comment:
I think we should drop using the the user's local Babel version. Instead we should just document that to use npm@3 or `npm deduplicate`. The user can then just enforce any Babel 6 version and we will use the same since our dependency is loose.